harness-mplr/harness.rb

```diff
@@ -26,5 +26,5 @@ def run_benchmark(_num_itrs_hint)
     total_time += time
   end until num_itrs >= MAX_BENCH_ITRS || total_time >= MAX_BENCH_SECONDS
 
-  return_results(times)
+  return_results(times, 15)
```

Comment:
I'm unsure how many iterations should be considered warmup for harness-mplr.
There is no `WARMUP_ITRS` env var or so.

Reply:
In general I think considering half the number of iterations (`times.size/2`) to be warmup is a good default.